internal/types/array.go

```diff
@@ -111,3 +111,33 @@ func (a *Array) Append(values ...any) error {
 func (a *Array) RemoveByPath(path Path) {
 	removeByPath(a, path)
 }
+
+// Contains checks if the Array contains the given value.
+func (a *Array) Contains(value any) bool {
+	for _, v := range a.s {
+		if Compare(v, value) == Equal {
+			return true
+		}
+	}
+	return false
+}
+
+// ContainsAll checks if the Array contains all the given values of the Array b.
+// Currently, this algorithm is O(n^2) without any performance tuning.
+// Possible improvement: to sorts both arrays before comparing.
```

Comment:
I think we should document that either inside the function or outside of it, but not in both places that we would need to keep in sync

Reply:
Moved everything outside of the function.